data/presets/building/waste_shed.json

```diff
@@ -0,0 +1,27 @@
+{
+    "icon": "maki-recycling",
+    "fields": [
+        "{building}",
+        "lockable",
+        "access"
+    ],
+    "geometry": [
+        "area"
+    ],
+    "locationSet": {
+        "include": ["DE", "EE", "SE"]
```

Comment:
exist also on Poland, though I mapped some with separate objects for shed, for group of recycling containers and for waste disposal

sometimes there are separate sheds for recyclables and general waste

Reply:
Maybe the `locationSet` property isn't needed.